arbisoft_sessions_portal/settings/base.py

```diff
@@ -149,6 +149,10 @@
 
 STATIC_URL = 'static/'
 
+# Media files
+MEDIA_URL = '/media/'
+MEDIA_ROOT = BASE_DIR / 'media'
+
```

Comment:
Since we will also store thumbnails under this path, should we name it `asset` instead? What do you think?

Reply:
Currently, since we are storing videos and thumbnails (which are images) uploaded by users, I think the name `media` works well as it aligns with the typical convention in Django projects for user-uploaded content.

If we plan to store other file types in the future, renaming the folder to `assets` would make sense to allow for broader categories and better reflect its purpose.